public/manageclaims.php

```diff
@@ -0,0 +1,199 @@
+        echo "<th>Expiration / Completion / Drop Date</th>";
+        echo "<th>Update</th>";
+
+        $userCount = 0;
+        foreach ($claimData as $claim) {
+            $claimID = $claim['ID'];
+            $claimUser = $claim['User'];
+            echo "<tr><td class='text-nowrap'>";
+            echo GetUserAndTooltipDiv($claimUser, true);
+            echo "</td>";
+            echo "<td class='text-nowrap'><div class='fixheightcell' id='claimUser_$claimUser'>";
+            echo GetUserAndTooltipDiv($claimUser, false);
+            echo "</div></td>";
+
+            echo "<td>";
+            echo "<select id='claimType_$claimID'>";
+            echo "<option " . ($claim['ClaimType'] == 0 ? "selected" : "") . " value='0'>Primary</option>";
+            echo "<option " . ($claim['ClaimType'] == 0 ? "" : "selected") . " value='1'>Collaboration</option>";
+            echo "</select>";
+            echo "</td>";
+
+            echo "<td>";
+            echo "<select id='setType_$claimID'>";
+            echo "<option " . ($claim['SetType'] == 0 ? "selected" : "") . " value='0'>New</option>";
+            echo "<option " . ($claim['SetType'] == 0 ? "" : "selected") . " value='1'>Revision</option>";
+            echo "</select>";
+            echo "</td>";
+
+            echo "<td>";
+            echo "<select id='status_$claimID'>";
+            switch ($claim['Status']) {
+                case 0:
+                    echo "<option selected value='0'>Active</option>";
+                    echo "<option value='1'>Complete</option>";
+                    echo "<option value='2'>Dropped</option>";
+                    break;
+                case 1:
+                    echo "<option value='0'>Active</option>";
+                    echo "<option selected value='1'>Complete</option>";
+                    echo "<option value='2'>Dropped</option>";
+                    break;
+                case 2:
+                    echo "<option value='0'>Active</option>";
+                    echo "<option value='1'>Complete</option>";
+                    echo "<option selected value='2'>Dropped</option>";
+                    break;
+                default:
+                    echo "<option selected value='0'>Active</option>";
+                    echo "<option value='1'>Complete</option>";
+                    echo "<option value='2'>Dropped</option>";
+                    break;
+            }
+            echo "</select>";
+            echo "</td>";
+
+            echo "<td>";
+            echo "<select id='special_$claimID'>";
+            echo "<option " . ($claim['Special'] == 0 ? "selected" : "") . " value='0'>0</option>";
+            echo "<option " . ($claim['Special'] == 1 ? "selected" : "") . " value='1'>1</option>";
+            echo "<option " . ($claim['Special'] == 2 ? "selected" : "") . " value='2'>2</option>";
+            echo "<option " . ($claim['Special'] == 3 ? "selected" : "") . " value='3'>3</option>";
```

Comment:
If there some reason not to use strings here?
* None
* Own Revision
* Free Rollout
* Release Scheduled